gyp/pylib/gyp/generator/msvs.py

```diff
@@ -174,6 +174,16 @@ def _FixPath(path):
   return path
 
 
+def _IsWindowsAbsPath(path):
+  """
+  On Cygwim systems Python needs a little help determining if a path is an absolute Windows path or not, so that
```

Comment:
```suggestion
  On Cygwin systems Python needs a little help determining if a path is an absolute Windows path or not, so that
```

Same typo also in commit message.